messages/descriptions/platform.json

```
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Platform description",
+    "description": "A description of a platform including metadata of the platform itself, available sensors and produced observations",
+    "required": [
+        "platform_id",
+        "metadata",
+        "sensors"
+    ],
+    "properties": {
+        "platform_id": {
+            "type": "string",
+            "title": "Platform id",
+            "desciption": "Id of the platform"
+        },
+        "metadata": {
+            "type": "object",
+            "title": "Metadata",
+            "description": "Metadata of this platform",
+            "required": [
+                "length",
+                "breadth"
+            ],
+            "properties": {
+                "length": {
+                    "type": "number",
+                    "title": "Length",
+                    "description": "Physical length of the platform"
+                },
+                "breadth": {
+                    "type": "number",
+                    "title": "Breadth",
+                    "description": "Physical breadth of the platform"
+                }
+            }
+        },
+        "sensors": {
+            "type": "array",
+            "title": "Sensors",
+            "description": "List of sensors associated with this platform",
+            "items": {
+                "$ref": "../descriptions/sensor.json"
+            }
+        }
```

I Shouldn't we add actuators here as well?

```suggestion
        },
        "actuators": {
            "type": "array",
            "title": "Actuators",
            "description": "List of actuators associated with this platform",
            "items": {
                "$ref": "../descriptions/actuator.json"
            }
        }
```

---

Most probably yes, but I think we are in a better position later on when we actually have a hands-on usecase so I suggest we wait with that.